src/View/Sidebar/BookmarkRow.vala

```diff
@@ -102,13 +102,14 @@ public class Sidebar.BookmarkRow : Gtk.ListBoxRow, SidebarItemInterface {
         var label = new Gtk.Label (custom_name) {
             xalign = 0.0f,
             halign = Gtk.Align.START,
-            hexpand = true,
-            margin_start = 6
```

Comment:
it's better to use `column_spacing` in the grid so that don't have to add margins to each child

Reply:
I think that may impact drag and dropping rows (the underlying list box would need to be made a drop target as well as the rows) but that may actually be cleaner